src/components/input-field/input-field.tsx

```diff
@@ -24,6 +24,11 @@ import {
 import { InputType } from './input-field.types';
 import { ListItem } from '@limetech/lime-elements';
 
+interface linkProperties {
```

Comment:
Interfaces usually start with a capital letter
```suggestion
interface LinkProperties {
```

Reply:
⚡